src/components/Balance.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import Promise from 'bluebird';
 import * as ethers from 'ethers';
 import { makeStyles } from '@material-ui/core/styles';
-import ERC20_CONTRACT_ABI from 'abis/erc20.json';
-import { formatUnits } from 'utils/big-number';
-import { useWallet } from 'contexts/wallet';
-import sleep from 'utils/sleep';
+import ERC20_CONTRACT_ABI from '../abis/erc20.json';
```

Comment:
Hey! I'd cp .env.sample to .env to have this https://github.com/vbstreetz/snx-migrate/blob/main/.env.sample#L1 but I guess we can also resolve this in a jsconfig.json.